app/lib/core/services/database/database_interface.dart

```diff
@@ -21,4 +21,6 @@ abstract class DatabaseInterface {
   Stream<List<AnimalTransportRecord>> getUpdatedCompleteATRs(String userId);
 
   Stream<List<AnimalTransportRecord>> getUpdatedActiveATRs(String userId);
+
+  Stream<Transporter> getUserInRealTime(String userId);
```

Comment:
Since the User and Transporter are separate entities, this would be better as:
```suggestion
  Stream<Transporter> getUpdatedTransporter(String userId);
```
The database interface is separate from the logic of the app.